kytos/core/controller.py

```diff
@@ -33,6 +33,8 @@
 
 __all__ = ('Controller',)
 
+LOCK_PATH = '/tmp/lockytos'
```

Comment:
This should be configured on `/etc/kytos/kytos.conf`. And should have a default value to `/var/run/kytos.pid`